### PR TITLE
Issue #5876: update xdoc for cmdline suppression option

### PR DESCRIPTION
**Pull request by romani**
Issue #5876

image

### CODE REVIEW

File: src/xdocs/cmdline.xml.vm [hunk 1]
```diff
@@ -80,9 +80,14 @@ java -D&lt;property&gt;=&lt;value&gt;  \
         </li>
         <li>
           <code>-s line:column</code> - print xpath suppressions at the file's line and column
-          position. Line and column numbers are integer values separated by a :, specify
+          position. Line and column numbers are integer values separated by a <code>:</code>,
+          specify
           location in the file the suppression should be generated for. The option cannot be used
           with other options and requires exactly one file to run on to be specified.
+          ATTENTION: generated result will have few queries, joined by pipe(<code>|</code>),
+          together they will match all AST nodes on specified line
```
[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs/cmdline.xml.vm]: done

File: src/xdocs/cmdline.xml.vm [hunk 2]
```diff
@@ -80,9 +80,14 @@ java -D&lt;property&gt;=&lt;value&gt;  \
...
+          and column, you need to choose only one and recheck that it works. Usage of all of them is
```
[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs/cmdline.xml.vm]: done.

File: src/xdocs/cmdline.xml.vm [hunk 3]
```diff
@@ -80,9 +80,14 @@ java -D&lt;property&gt;=&lt;value&gt;  \
...
+          also ok, but might result on non-desireable matching and suppress another issues.
```
[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs/cmdline.xml.vm]: done.

**Comment by rnveach** [comment]
ok to merge if CI passes.

**Comment by romani** [comment]
squashed and merged manually

**Pull request closed without merging**